data/data/kubevirt/main.tf

```diff
@@ -0,0 +1,68 @@
+provider "kubernetes" {
+  config_path = "/home/nargaman/.kube/config"
+}
+
+provider "kubernetes-alpha" {
+  server_side_planning = false
+  config_path = "/home/nargaman/.kube/config"
```

Comment:
change the path to be relative for the running directory

Reply:
Done